data-prepper-plugins/dynamodb-source/src/main/java/org/opensearch/dataprepper/plugins/source/dynamodb/converter/MetadataKeyAttributes.java

```diff
@@ -12,9 +12,11 @@ public class MetadataKeyAttributes {
 
     static final String SORT_KEY_METADATA_ATTRIBUTE = "sort_key";
 
-    static final String EVENT_TIMESTAMP_METADATA_ATTRIBUTE = "ts";
+    static final String EVENT_TIMESTAMP_METADATA_ATTRIBUTE = "ddb_ts";
```

Comment:
Why do we shorted the name at all? That is, why not name this `ddb_timestamp`? If somebody looks at this metadata, it would not be clear what `ddb_ts` means.

Reply:
I agree. I'll update.